src/shared/components/ui/skeleton/SkeletonCardLoader.tsx

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from '@tanstack/react-query';
+import { SkeletonCard } from './SkeletonCard';
+
+interface SkeletonCardLoaderProps {
+  queryKey: any[];
+  queryFn: () => Promise<any>;
+  variant: 'list' | 'recommend' | 'detailTop' | 'detailList';
+  render: (data: any) => React.ReactNode;
+}
```

Comment:
`any` 타입은 가급적이면 지양하시는 게 좋습니다! Generic 또는 `unknown`으로 구성하시는 것을 권장드립니다.

```suggestion
interface SkeletonCardLoaderProps<T> {
  queryKey: readonly unknown[]; // queryKey는 readonly 권장
  queryFn: () => Promise<T>;
  variant: 'list' | 'recommend' | 'detailTop' | 'detailList';
  render: (data: T) => React.ReactNode;
}
```